README.md

```diff
@@ -58,37 +58,48 @@ Create and deploy an app-operator using the SDK CLI:
 ```sh
 # Create an app-operator project that defines the App CR.
 $ mkdir -p $GOPATH/src/github.com/example-inc/
+# Create a new app-operator project
 $ cd $GOPATH/src/github.com/example-inc/
-$ operator-sdk new app-operator --api-version=app.example.com/v1alpha1 --kind=App
+$ operator-sdk new app-operator
 $ cd app-operator
 
+# Add a new API for the custom resource AppService
+$ operator-sdk add api --api-version=app.example.com/v1alpha1 --kind=AppService
+
+# Add a new controller that watches for AppService
+$ operator-sdk add controller --api-version=app.example.com/v1alpha1 --kind=AppService
+
 # Build and push the app-operator image to a public registry such as quay.io
 $ operator-sdk build quay.io/example/app-operator
 $ docker push quay.io/example/app-operator
 
 # Update the operator manifest to use the built image name
 $ sed -i 's|REPLACE_IMAGE|quay.io/example/app-operator|g' deploy/operator.yaml
 
+# Setup RBAC
+$ kubectl create -f deploy/role.yaml
+$ kubectl create -f deploy/role_binding.yaml
+# TODO: kubectl create -f deploy/service_account.yaml
+# Setup the CRD
+$ kubectl create -f deploy/crds/app_v1alpha1_appservice_crd.yaml
 # Deploy the app-operator
-$ kubectl create -f deploy/sa.yaml
-$ kubectl create -f deploy/rbac.yaml
-$ kubectl create -f deploy/crd.yaml
 $ kubectl create -f deploy/operator.yaml
 
-# By default, creating a custom resource (App) triggers the app-operator to deploy a busybox pod
-$ kubectl create -f deploy/cr.yaml
+# Create an AppService CR
+# The default controller will watch for AppService objects and create a pod for each CR
+$ kubectl create -f deploy/crds/app_v1alpha1_appservice_cr.yaml
 
-# Verify that the busybox pod is created
-$ kubectl get pod -l app=busy-box
-NAME            READY     STATUS    RESTARTS   AGE
-busy-box   1/1       Running   0          50s
+# Verify that a pod is created
+$ kubectl get pod -l app=example-appservice
+NAME                     READY     STATUS    RESTARTS   AGE
+example-appservice-pod   1/1       Running   0          1m
 
 # Cleanup
-$ kubectl delete -f deploy/cr.yaml
-$ kubectl delete -f deploy/crd.yaml
+$ kubectl delete -f deploy/app_v1alpha1_appservice_cr.yaml
 $ kubectl delete -f deploy/operator.yaml
-$ kubectl delete -f deploy/rbac.yaml
-$ kubectl delete -f deploy/sa.yaml
+$ kubectl delete -f deploy/role.yaml
+$ kubectl delete -f deploy/role_binding.yaml
+$ kubectl delete -f deploy/app_v1alpha1_appservice_crd.yaml
```

Comment:
This should be `kubectl delete -f  deploy/crds/app_v1alpha1_appservice_crd.yaml `